registry.json

```diff
@@ -2939,5 +2939,13 @@
     "telegram": "FreedomSoCal",
     "operatorName": "Rory",
     "github": "Hdpro4u"
+  },
+     {
+    "ticker": "PRIDE",
+    "poolId": "93ca5c2f1eb63054f7c768a807408a43e65d7c68fe56e330db20af51",
+    "social": {
```

Comment:
You need to have the other social media entries even if they are blank, e.g. reddit, facebook etc. Please add the missing keys from social.